tools/deployer/migrations/1_initial_migration.js

```diff
@@ -67,14 +89,22 @@ module.exports = async (deployer, network) => {
   await Validator.link('Types', Types.address)
   await deployer.deploy(Validator, WETH_ADDRESS)
 
-  //Deploy Contract-Created Contracts that also need to be verified
-  await deployer.deploy(Index)
-  await deployer.deploy(
-    Delegate,
-    Swap.address,
-    Indexer.address,
-    ADDRESS_ZERO,
-    ADDRESS_ZERO,
-    '0x0001'
-  )
+  //Deploy Factory-Created Contracts that also need to be verified
+  if (network !== "DEVELOPMENT") {
```

Comment:
Could merge this and the following conditional. Under what circumstances will network === 'DEVELOPMENT'?

Reply:
> Under what circumstances will network === 'DEVELOPMENT'?
Localhost